Makefile

```diff
@@ -32,6 +32,9 @@ check: test
 	# tests should fail if go fmt results in uncommitted code
 	git status --porcelain
 	/bin/bash -c 'test $$(git status --porcelain | wc -l) == 0'
+	go get github.com/gordonklaus/ineffassign
+	"${GOPATH}/bin/ineffassign" .
```

Comment:
let's use metalinter instead:
https://github.com/alecthomas/gometalinter

Then we can specify the rules we want to run...